### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MoonkiHong**
Signed-off-by: MoonkiHong <moonki1.hong@samsung.com>

Representing the security vulnerability validation status from README.md in the root. (w.r.t. the issue #141)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code cleanup/refactoring
- [x] Documentation update
- [ ] This change requires a documentation update

# How Has This Been Tested?
This is a document update, so I did not locally test it, but validated successfully the build test given from the GitHub Action integrated into this project.

**Test Configuration**:
* Firmware version: (OS type and version, e.g., Ubuntu 14.04, etc.)
* Hardware: (e.g., x86-64, ARM, CPU type, RAM size, etc.)
* Toolchain: (e.g., Docker and Go versions)
* Edge Orchestration Release: (e.g., Alpha, Baobab, Coconut)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Edge Orchestration
 
 [![Goreport](https://goreportcard.com/badge/github.com/lf-edge/edge-home-orchestration-go)](https://goreportcard.com/report/github.com/lf-edge/edge-home-orchestration-go)
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/lf-edge/edge-home-orchestration-go.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/lf-edge/edge-home-orchestration-go/alerts/)
```
Comment (t25kim) [inline README.md:4]: Please add the actions badge as well.
https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge

**Review by t25kim — COMMENTED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (MoonkiHong) [reply README.md:4]: I will do that!! Thank you for your suggestion!!! :-)

Comment (MoonkiHong) [reply README.md:5]: Applied complete. PTAL. ^^

**Pull request merged**